Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1322,6 +1344,11 @@ private extension Purchases {
         self.offeringsManager.updateOfferingsCache(appUserID: self.appUserID,
                                                    isAppBackgrounded: isAppBackgrounded,
                                                    completion: nil)
+
+        self.offlineEntitlementsManager.updateProductsEntitlementsCacheIfStale(
+            isAppBackgrounded: isAppBackgrounded,
+            completion: nil
+        )
```

Comment:
We should hold off merging this until the backend is ready. I guess if not we will just get some errors...

Reply:
Yup.